src/ai/backend/agent/resources.py

```diff
@@ -428,6 +428,13 @@ def get_additional_gids(self) -> list[int]:
         """
         return []
 
+    def get_additional_allowed_syscalls(self) -> list[str]:
+        """
+        Returns system calls allowed within the container.
+        These system calls will be additionally allowed in addition to those allowed by the default seccomp profile.
```

Comment:
It will be good if we can make a reference to the examples here, either as a string or a URL of syscall names.